http/http_service.go

```diff
@@ -285,6 +294,17 @@ func (httpSvc *HttpService) startHandler(c echo.Context) error {
 		})
 	}
 
+	// NOTE: the config is also unlocked as part of the start
+	// goroutine below. But since we execute start asynchronously it's hard to
+	// know when the config has been unlocked before being able to create the JWT token
+	err := httpSvc.cfg.Unlock(startRequest.UnlockPassword)
```

Comment:
so it runs twice potentially in parallel?

Reply:
No, it's not in parallel - this first call here is synchronous